pkg/thing/handler/amqp/msg_handler.go

```diff
@@ -79,6 +79,16 @@ func (mc *MsgHandler) handleRegisterMsg(body []byte, authorizationHeader string)
 	return mc.thingInteractor.Register(authorizationHeader, msgParsed.ID, msgParsed.Name)
 }
 
+func (mc *MsgHandler) handleUnregisterMsg(body []byte, authorizationHeader string) error {
+	msgParsed := network.UnregisterRequestMsg{}
```

Comment:
As the function is small, I think this variable could be renamed to `msg`.

Reply:
@joaoaneto Agree with you, done!